knowledge-base/ad-m/github-push-action/action-security.yml

```diff
@@ -0,0 +1,10 @@
+name: 'GitHub Push'
+github-token:
+  action-input:
+    input: github_token
+    is-default: true
+  permissions:
+    contents: write
+    contents-reason: to push changes local changes  #Checkout: https://github.com/ad-m/github-push-action#github-action-for-github-push 
```

Comment:
@h0x0er can you please change the reason to `to push local changes`? changes is getting repeated.